src/Microsoft.SqlTools.Dmp.Hosting/Protocol/MessageReader.cs

```diff
@@ -0,0 +1,262 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.SqlTools.Dmp.Hosting.Utility;
+
+namespace Microsoft.SqlTools.Dmp.Hosting.Protocol
+{
+    public class MessageReader
```

Comment:
It's a shame we don't have this tracked as a simple code move. I'm going to assume that other than the namespace change everything here is unchanged and ignore it.

Reply:
There's some changes since the serialization/deserialization code has moved around a bit 
https://www.diffchecker.com/kYRRQ7bj